Resources/Private/Templates/Haiku/Show.html

```diff
@@ -0,0 +1,24 @@
+<html data-namespace-typo3-fluid="true"
+      xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers">
+
+<f:layout name="Default"/>
```

Comment:
You use layout but haven't defined one? I suggest to add a layout which just renders the template.

Reply:
I am using the Layout which has already been there